.github/workflows/stage.yml

```diff
@@ -5,30 +5,42 @@ on:
   workflow_dispatch:
   push:
     branches:
-      - main
+      - master
     paths-ignore:
       - ".github/**"
 
 jobs:
   build:
     runs-on: ubuntu-latest
+    if: github.repository_owner == 'fabric8-analytics'
     name: Build plugin
+
     steps:
-      - name: Checkout sources
+      - name: Checkout repository
         uses: actions/checkout@v3
-
-      - name: Setup Node
-        uses: actions/setup-node@v2.1.2
         with:
-          node-version: 14.x
+          ssh-key: ${{ secrets.DEPLOY_KEY }}
+      
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          registry-url: 'https://npm.pkg.github.com'
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Create .npmrc
+        run: |
+          echo "@RHEcosystemAppEng:registry=https://npm.pkg.github.com" >> ~/.npmrc
+          echo "@fabric8-analytics:registry=https://npm.pkg.github.com" > ~/.npmrc
+      
       - name: Install @vscode/vsce
         run: npm i -g @vscode/vsce
 
       - name: Install Dependencies
         run: npm ci
 
-      - name: Compile
+      - name: Compile for test
```

Comment:
@IlonaShishov Is this working?? i  remember that we had problems with compildation for tests didn't we?

Reply:
yes, 
we are using a workaround for the incompatibility between CommonJS and ESM modules and temporarily not testing sections of code invoking the exhort-javascript-api dependency.